optimum/intel/openvino/quantization.py

```diff
@@ -347,8 +347,12 @@ def _quantize_ovbasemodel(
                     remove_unused_columns=remove_unused_columns,
                     data_collator=data_collator,
                 )
-
                 if self.model.export_feature == "text-generation" and self.model.use_cache:
+                    if self.model.stateful:
+                        raise ValueError(
+                            "Currently, only non-stateful models are supported for quantization, to export such model you can do : `.from_pretrained(export=True, stateful=False)`"
+                        )
+
```

Comment:
Could you please explain why stateful models are not supported? On my side the test for `gpt2` model with removed `stateful=False`, quantizes model without any errors.

Reply:
I had issues with it when testing but it looks like it's working now, added it back in https://github.com/huggingface/optimum-intel/pull/727/commits/e7f1569f37a84077b1fb4532ff6f40445f812724